File: src/main/java/gay/skitbet/jackiro/utils/JackiroConfig.java
package gay.skitbet.jackiro.utils;

import lombok.*;
import lombok.experimental.Accessors;

import java.util.ArrayList;
import java.util.List;

@NoArgsConstructor
@Data
@Accessors(chain = true)
public class JackiroConfig {

    public String clientToken;
    public String clientID;
    public String clientSecret;
    public String ownerId;
    public int shardCount;
    public List<String> currentlyPlaying = new ArrayList<>();


}

File: src/main/kotlin/gay/skitbet/jackiro/utils/JackiroConfig.kt
package gay.skitbet.jackiro.utils

data class JackiroConfig(
    var clientToken: String? = null,
    var clientID: String? = null,
    var clientSecret: String? = null,
    var ownerId: String? = null,
    var shardCount: Int = 0,
    var currentlyPlaying: MutableList<String> = mutableListOf()
)
